packages/@okta/vuepress-site/guides/sign-into-spa/section09/angular/allowanon.md

```diff
@@ -1 +1 @@
-placeholder
\ No newline at end of file
+By default, all routes without `OktaAuthGuard` will be accessible to unauthenticated users.
```

Comment:
By default, all routes without `OktaAuthGuard` are accessible to unauthenticated users.